routes/rooms.js

```diff
@@ -0,0 +1,36 @@
+const express = require('express');
+const router = express.Router();
+const MongoClient = require('mongodb').MongoClient;
+
+const mongoUri = process.env.MONGODB_URI;
+const roomsCollection = "wye-rooms";
+
+/* GET users listing. */
+router.get('/', function(req, res, next) {
+  MongoClient.connect(mongoUri, function(err, db) {
```

Comment:
I guess you planned to move to something like https://mongodb.github.io/node-mongodb-native/driver-articles/mongoclient.html#mongoclient-connection-pooling for the db initialisation?

Sounds completely fine for a first usage. We will update it in another MR.

Reply:
Yes but we didn't choose the driver yet so I took one.
I'll add an issue to be sure to not forget

Reply:
`mongoose` seems nice for the `Schema` feature.